### DISCUSSION TITLE
Firebase + Apple's App Privacy Report / Privacy Label

### DISCUSSION THREAD

**Discussion by SwiftNativeDeveloper** [firebase/firebase-ios-sdk#9004] [General]
Hey Firebase team and community,

Apple is releasing their new App Privacy Report UI in iOS 15.2. https://developer.apple.com/news/?id=n5jlz7ox
In summary, it informs users what websites apps communicate with and what sensitive information it accesses. 

Is there any documentation that breaks down which Firebase components talk to what domains/subdomains and access which of the 'sensitive info' categories that Apple shows in their report? The idea behind this ask is so that we can accurately describe to users why they see domains that don't match our brand name.

Same question for the App Privacy Label found in App Store. Which Firebase components "collect" what privacy label categories of data (apple's term for transmitting off device)?

**Comment by morganchen12** [comment]
Hey @SwiftNativeDeveloper, you can find our docs for the app privacy labels [here](https://firebase.google.com/docs/ios/app-store-data-collection).

We currently don't have plans to publicly document the endpoints of individual Firebase services, but if you or other devs start seeing user feedback asking about the urls in their apps' settings pages we certainly can.

**Comment by godot007** [comment]
In order to reduce the instability caused by SDK upgrade, we hope not to upgrade the SDK for the time being and integrate the privacy manifests into the main app project. Which item of 85F4.1/E174.1/7D9E.1/B728.1 should be filled in for the NSPrivacyAccessedAPICategoryDiskSpace involved in FirebaseCrashlytics (8.15.0)?
